src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormImages.tsx

```diff
@@ -125,6 +128,20 @@ export const MyCollectionArtworkFormImages = forwardRef<
     const imageAlreadyAdded = localImages.find(
       localImage => localImage.photoID === photoID
     )
+
+    // Handle images added through the select artwork step
+    if (!isEditing && currentSrc.includes("cloudfront.net")) {
```

Comment:
I am using `isEditing` here to make this only happens when the user is adding an artwork for the first time to avoid duplicates